vnext/local-cli/generator-windows/templates/cpp/src/App.cpp

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
is this right? I don't think we want to claim license of the app's code?

Reply:
Hmm yeah, looks like we don't want license text in package provider either as @NickGerleman said. I'll close this PR😬